### PR TITLE
Add Master Pasword app (latest non-MAS version)

### PR DESCRIPTION
**Pull request by osteotek**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/master-password.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  url 'https://ssl.masterpasswordapp.com/masterpassword-mac.zip'
+  name 'Master Password'
+  homepage 'http://masterpasswordapp.com'
+  license :gpl
+
+  app 'Master Password.app'
+
+  uninstall quit:      'com.lyndir.lhunath.MasterPassword.Mac',
+            launchctl: 'com.lyndir.lhunath.MasterPassword.Mac.LoginHelper',
+            pkgutil:   'com.lyndir.lhunath.MasterPassword.*',
```
Comment (reitermarkus) [inline Casks/master-password.rb]: As there is no `.pkg` used to install the application, is this really needed?

Comment (osteotek) [inline Casks/master-password.rb]: Yeah, I misunderstood the purpose of this stanza. Let me update

File: Casks/master-password.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            delete:    '/Applications/Master Password.app'
```
Comment (reitermarkus) [inline Casks/master-password.rb]: This also needs to be removed, as this is done automatically.

Comment (osteotek) [inline Casks/master-password.rb]: Will do

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 340036d73ac9b188716d7eab656dd5b84fb2277d. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**